packages/world/index.ts

```diff
@@ -248,12 +260,12 @@ export namespace RegionReader {
      * @param index The chunk index
      */
     export function seekBlockStateId(section: NewRegionSectionDataFrame | LegacyRegionSectionDataFrame, index: ChunkIndex) {
-        if ("BlockStates" in section) {
-            const blockStates = section.BlockStates;
-            const bitLen = computeBitLen(section.Palette, blockStates);
-            return seek(blockStates, bitLen, index);
+        if ("Blocks" in section) {
```

Comment:
No worries I should have been more clear.

When it detects a new chunk format (not the legacy one) it uses `getSectionBlockIdArray` to get the block id array (variable was previously called vector).

I have also marked this function as exported so you can choose to use this directly.

This should be a non-breaking change that both maintains backwards support with older versions, fixes the chunk corruption and adds the new exported function `getSectionBlockIdArray` useable on sections with a non-legacy format.

Please let me know if this doesn't make sense.

Reply:
> No worries I should have been more clear.
> 
> When it detects a new chunk format (not the legacy one) it uses `getSectionBlockIdArray` to get the block id array (variable was previously called vector).
> 
> I have also marked this function as exported so you can choose to use this directly.
> 
> This should be a non-breaking change that both maintains backwards support with older versions, fixes the chunk corruption and adds the new exported function `getSectionBlockIdArray` useable on sections with a non-legacy format.
> 
> Please let me know if this doesn't make sense.

Sure, let's update the branch with master and run test for it. As long as the old tests for 1.12.2 and 1.14.4 don't fail, it will be totally ok for me.